src/DataTable.php

```diff
@@ -800,6 +851,52 @@ public function buildSearch(): Builder
             }
         }
 
+        // include selected relationships
+        $tmpWith = [];
+        $with = [];
+        $loadedModels = [];
+        $baseModelInfo = ModelInfo::forModel($this->model);
+        $modelQuery = new $this->model;
+        foreach ($this->enabledCols as $enabledCol) {
+            $foreignKeys = [];
+            if (str_contains($enabledCol, '.')) {
+                $explodedCol = explode('.', $enabledCol);
+                $attribute = Arr::pull($explodedCol, count($explodedCol) - 1);
```

Comment:
isn't this just an array_pop(). $attribute = array_pop($explodedCol);

Reply:
i would prefer keeping the Arr::pull because i already have the Arr helper for other points included